src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -167,4 +174,33 @@ public ResponseEntity<Void> stopBuild(@Parameter(description = "Build receiver")
         networkModificationService.stopBuildRequest(receiver);
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping(value = "/network-modifications/line/catalog", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get a line catalog")
```

Comment:
Get the catalog lines or lines of the catalog

Reply:
fixed, I chose :
"Get line types catalog" etc...